stdlib/public/SDK/CoreGraphics/CGFloat.swift.gyb

```diff
@@ -807,3 +807,17 @@ extension CGFloat : _CVarArgPassedAsDouble, _CVarArgAligned {
     return native._cVarArgAlignment
   }
 }
+
+extension CGFloat : Codable {
+  @_transparent
+  public init(from decoder: Decoder) throws {
+    let container = try decoder.singleValueContainer()
+    self.native = try container.decode(NativeType.self)
```

Comment:
Yeah, that seems like a reasonable middle ground. I guess I was assuming that the `Int` case would be okay because `Encoder`s and `Decoder`s need to natively handle `Int` already, but that's not true for `CGFloat`.